representatives/models.py

```diff
@@ -130,6 +125,31 @@ def gender_as_str(self):
         genders = {0: 'N/A', 1: 'F', 2: 'M'}
         return genders[self.gender]
 
+    def get_country(self):
+        """
+        Get the representative country.
+
+        Note that this will butcher your database if you didn't get the
+        representatives from ``with_countries()``.
+        """
+        for m in self.mandates.all():
+            if not m.constituency.country_id:
+                continue
+
+            return m.constituency.country
+
+    def get_current_mandate(self):
+        today = date.today()
+
+        for m in self.mandates.all():
+            if m.end_date < today:
+                continue
+
+            if m.group.kind != 'group':
+                continue
+
+            return m
+
```

Comment:
If you do it in SQL, you'll do one SQL query per Representative you call. Instead, we leverage prefetch_related, implemented here (as you can see it does the select related in the prefetch to have mandate.constituency.country and group):

https://github.com/political-memory/django-representatives/pull/12/files#diff-375065bb5decf79fe0128018a8a8ecafR88

and tested here (see the assertNumQueries):

https://github.com/political-memory/django-representatives/pull/12/files#diff-77278a91b6628e42e6059a58b56a466bR19